uxarray/grid/grid.py

```diff
@@ -804,8 +804,14 @@ def face_areas(self) -> np.ndarray:
             self._face_areas, self._face_jacobian = self.compute_face_areas()
         return self._face_areas
 
-    # ==================================================================================================================
+    @property
+    def bounds(self):
+        """Declare bounds as a property."""
+        if self._bounds is None:
+            self._bounds = _populate_bounds(self)
+        return self._bounds
```

Comment:
May you add a description of what `bounds` is here, something similar in style to the existing ones would be good (attached what we have for face nodes as a reference).




https://github.com/UXARRAY/uxarray/blob/d3a4c5f572b0f716c29165fd83c2def2cb347dec/uxarray/grid/grid.py#L668C5-L676